### PR TITLE
[PF-303] Following RBS integration, remove the old Google project creation code.

### PR DESCRIPTION
**Pull request by tnadler**

### CODE REVIEW

**Review by wchamber — APPROVED** [review]

**Review by zloery — APPROVED** [review]

File: src/test/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextRBSFlightTest.java [hunk 1]
```diff
@@ -75,7 +79,7 @@ public void successCreatesProjectAndContext() throws Exception {
         workspaceService.getCloudContext(workspaceId, userReq));
     Project project = crl.getCloudResourceManagerCow().projects().get(projectId).execute();
     assertEquals(projectId, project.getProjectId());
-    assertServiceApisEnabled(project, CreateProjectStep.ENABLED_SERVICES);
+    assertServiceApisEnabled(project, ENABLED_SERVICES);
```
[the remark this answers is not in the document]
Comment (tnadler) [reply src/test/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextRBSFlightTest.java:82]: Yeah, I mostly left it as a sanity check, but happy to remove.

File: src/test/java/bio/terra/workspace/service/workspace/flight/DeleteGoogleContextFlightTest.java [hunk 2]
```diff
@@ -53,7 +53,7 @@ public void deleteContext() throws Exception {
     FlightState flightState =
         StairwayTestUtils.blockUntilFlightCompletes(
             jobService.getStairway(),
-            CreateGoogleContextFlight.class,
+            CreateGoogleContextRBSFlight.class,
             createParameters,
             CREATION_FLIGHT_TIMEOUT);
```
[the remark this answers is not in the document]
Comment (tnadler) [reply src/test/java/bio/terra/workspace/service/workspace/flight/DeleteGoogleContextFlightTest.java:58]: Changed to 3 minutes.

File: src/main/java/bio/terra/workspace/app/configuration/external/BufferServiceConfiguration.java [hunk 3]
```diff
@@ -16,23 +16,13 @@
 @ConfigurationProperties(prefix = "workspace.buffer")
 public class BufferServiceConfiguration {
 
-  // TODO(PF-302): Clean up once fully using Buffer Service.
-  private boolean enabled = false;
```
[the remark this answers is not in the document]
Comment (tnadler) [reply src/main/java/bio/terra/workspace/app/configuration/external/BufferServiceConfiguration.java:20]: Good catch, thank you.

File: src/test/java/bio/terra/workspace/service/workspace/flight/DeleteGoogleContextFlightTest.java [hunk 4]
```diff
@@ -53,7 +53,7 @@ public void deleteContext() throws Exception {
     FlightState flightState =
         StairwayTestUtils.blockUntilFlightCompletes(
             jobService.getStairway(),
-            CreateGoogleContextFlight.class,
+            CreateGoogleContextRBSFlight.class,
```
[the remark this answers is not in the document]
Comment (tnadler) [reply src/test/java/bio/terra/workspace/service/workspace/flight/DeleteGoogleContextFlightTest.java:56]: I think I am just going to rename the new thing back to CreateGoogleContextFlight (without RBS). That is an implementation detail.

**Pull request merged**